### PR TITLE
Playwright/adding GitHub actions pipeline

### PR DESCRIPTION
**Pull request by MurilloDeOliveiraa**

### CODE REVIEW

File: E2E/FirstTest.spec.js [hunk 1]
```diff
@@ -7,5 +7,6 @@ test('My first Test', async ({ page }) => {
     await page.locator('#APjFqb').click();
     await page.locator('#APjFqb').fill('Globo');
     await page.keyboard.press('Enter');
-    await expect(page).toHaveTitle("Globo - Pesquisa Google");
+    await expect(page).toHaveTitle("Globo - Google Search");
+    // Test 2
```
Comment (murillo-QA) [inline E2E/FirstTest.spec.js]: We should remove this commented line

File: E2E/FirstTest.spec.js [hunk 2]
```diff
@@ -7,5 +7,6 @@ test('My first Test', async ({ page }) => {
     await page.locator('#APjFqb').click();
     await page.locator('#APjFqb').fill('Globo');
```
Comment (murillo-QA) [inline E2E/FirstTest.spec.js]: Put the locator in an variable and use this variable on lines 7 and 8

**Review by murillo-QA — CHANGES_REQUESTED** [review]
Small changes to do before merging but great job!

File: E2E/FirstTest.spec.js (hunk 2 above)
Comment (MurilloDeOliveiraa) [reply E2E/FirstTest.spec.js]: Done!

File: E2E/FirstTest.spec.js (hunk 1 above)
Comment (MurilloDeOliveiraa) [reply E2E/FirstTest.spec.js]: Done!

**Review by murillo-QA — APPROVED** [review]

**Pull request merged**